packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -174,23 +172,23 @@ export const RelationInputDataManager = ({
       labelLoadMore={
         !isCreatingEntry
           ? formatMessage({
-              id: getTrad('relation.loadMore'),
+              id: 'relation.loadMore',
```

Comment:
Imo we should keep it wrapped in `getTrad()` because these translations are bound to the content-manager.

Could you revert that part of the change and prefix + move the translations you have added into the `content-manager` namespace, please?

Reply:
indeed! thanks @gu-stav , fixed with e41226d